app/views/compute_resources/show/_ovirt.html.erb

```diff
@@ -2,3 +2,7 @@
   <td><%= _("URL") %></td>
   <td><%= @compute_resource.url %></td>
 </tr>
+<tr>
+  <td><%= _("Operating systems API supported?") %></td>
+  <td><%= @compute_resource.supports_operating_systems? ? _("Yes") : _("No") %></td>
```

Comment:
maybe some eyecandy to keep consistency, you can either use status icons that we have for hosts and proxies or bw icons we use at templates
![eye1](https://cloud.githubusercontent.com/assets/109773/15539580/9e344c1c-2283-11e6-894b-8feb2c8f077a.jpg)
![eye2](https://cloud.githubusercontent.com/assets/109773/15539581/9e38ac26-2283-11e6-88fa-1b408b7bffad.jpg)